packages/volto/src/components/manage/Toolbar/Toolbar.jsx

```diff
@@ -455,6 +459,7 @@ class Toolbar extends Component {
             </div>
           </div>
           <div
+            id="toolbar"
```

Comment:
I just realized the `toolbar` id already exists in the page: https://github.com/plone/volto/blob/main/packages/volto/src/helpers/Html/Html.jsx#L195

```suggestion
            id="toolbar-body"
```